src/index/repositories.props

```diff
@@ -28,7 +28,7 @@
 /*
       </SparseCheckout>
       <PrepareCommand>
-        src\coreclr\build.cmd -skiptests -skipnative -linuxmscorlib &amp;&amp; libraries.cmd /p:BuildNative=false /p:BuildPackages=false -binarylog -os Linux
+        build.cmd -subsetcategory coreclr-libraries -subset corelib-all /p:BuildNative=false /p:BuildPackages=false -binarylog -os Linux
```

Comment:
Will this build both corelib and the libraries? Is this going to build every version of corelib? It should only build the linux mscorlib, because we use existence of the output file to determine which version of a project to index.

Reply:
Because of the -os parameter at the end this builds Linux (only).

> Will this build both corelib and the libraries?
This builds CoreLib from coreclr and libraries managed code.

Reply:
Can we test the changes somehow? :)

Reply:
You can just run `msbuild build.proj` locally. It will take a long time, but will produce the index locally that you can run.

Reply:
I'm relatively certain this is good as is. Let's merge and I'll monitor the build.